lib/wallet/api/http/Cardano/Wallet/Api.hs

```diff
@@ -1240,26 +1242,27 @@ instance HasWorkerCtx (DBLayer IO s k) (ApiLayer s k ktype) where
 -------------------------------------------------------------------------------}
 
 type HasWorkerRegistry s k ctx =
```

Comment:
```suggestion
type HasWorkerRegistry s ctx =
```

Following Yura's suggestions, `k` can probably also be removed here? 🤔

(That said, at some point, it will be better to merge the pull request rather than to cover everything; we can always make a second pull request.)